sdcm/tester.py

```diff
@@ -885,6 +886,57 @@ def prepare_kms_host(self) -> None:
         self.params["append_scylla_yaml"] = append_scylla_yaml
         return None
 
+    def prepare_azure_kms(self) -> None:
+        scylla_version = self.params.scylla_version
+        if not scylla_version:
+            return None
+        version_supports_kms = ComparableScyllaVersion(self.params.scylla_version) >= '2025.4.0~dev'
```

Comment:
Why not using the short variant `scylla_version` which was defined on the line 890?
Or why defining the short variant then?